src/plugins/vcNarrator.tsx

```diff
@@ -165,7 +165,7 @@ export default definePlugin({
                 if (!type) continue;
 
                 const template = Settings.plugins.VcNarrator[type + "Message"];
-                const user = isMe ? "" : UserStore.getUser(userId).username;
+                const user = Settings.plugins.VcNarrator["narrateSelf"] && isMe ? "" : UserStore.getUser(userId).username;
```

Comment:
narrateSelf is kind of a misleading name and this if should be inverted
```suggestion
                const user = isMe && !Settings.plugins.VcNarrator.sayOwnName ? "" : UserStore.getUser(userId).username;
```